app/Http/Controllers/DocsController.php

```diff
@@ -25,6 +25,9 @@ public function __invoke(Documentation $docs, ?string $page = null)
             return redirect()->route('docs', [self::DEFAULT_PAGE]);
         }
 
+        $isMarkdown = str($page)->endsWith('.md');
+        $page = str($page)->replace('.md', '')->toString();
```

Comment:
i don't love that this mutates the original parameter but it seemed like a better call than refactoring all the existing usages of it given i couldn't run the tests locally